### PR TITLE
Provides better format for database connection errors

### LINKED ISSUE #511: Whishlist: A connection box when database failed.
In case of problem we got.

capture d ecran 2017-02-10 a 20 25 51

If we can replace this with a simple box with a message in this style.

error

### PR DESCRIPTION
**Pull request by asuweb**
Fixes #511 
image

### CODE REVIEW

File: mailscanner/database.php [hunk 1]
```diff
@@ -59,7 +59,30 @@ public static function connect($host = '', $username = '', $password = '', $data
...
+.db-error {
+    width: 40%;
+    margin: 0 auto;
+    text-align: center;
+    margin-top: 100px;
+    border: solid 5px #000;
+    -webkit-border-radius:20px;
+    -moz-border-radius:20px;
+    border-radius:20px;
+    background: rgb(255, 0, 0);
```
Comment (endelwar) [inline mailscanner/database.php]: what about a lighter red like #ff6961 ?

Comment (asuweb) [inline mailscanner/database.php]: @endelwar - I'm happy with whatever colour people prefer - best ask @stefaweb ;)

Comment (endelwar) [inline mailscanner/database.php]: schermata 2017-06-13 alle 19 10 06
```
    background-color: #f2dede;
    border: solid 3px #ebcccc;
    color: #a94442;
```

Comment (asuweb) [inline mailscanner/database.php]: Yes, that's good for me :)

**Pull request merged**